department-of-veterans-affairs/vets-website
# 51630 - Remove conditional use of the old isCerner method

## Summary
This PR removes the conditional use of the 'vaOnlineSchedulingUseDsot' feature flag.


## Related issue(s)
- department-of-veterans-affairs/va.gov-team#51630


## Testing done
Unit testing

**How to testing locally**
_Prerequisite:_ Make sure the [vamc-ehr.json](https://www.va.gov/data/cms/vamc-ehr.json) file exists under build/localhost/data/cms. If not, create the file and copy the content retrieved from visiting the following link: https://www.va.gov/data/cms/vamc-ehr.json into the file.

**NOTE:**
You may have to change the facility you want to test to a 'cerner' facility. Ex. Search for facility id '442', then change "fieldVamcEhrSystem": "vista" to "fieldVamcEhrSystem": "cerner". 

1. Modify './vaos/services/mocks/index.js:426' by changing 'facility_id' from '983' to '442'
2. Schedule an appointment
3. The 'Choose a VA location' page should look like screenshot


## Screenshots
![localhost_3001_health-care_schedule-view-va-appointments_appointments_new-appointment_va-facility-2](https://user-images.githubusercontent.com/3587066/213225049-3b37a522-2644-4645-b129-f1b0e81c6576.png)


## What areas of the site does it impact?
VAOS

## Acceptance criteria


## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
